lib/shared/messages.js

```diff
@@ -123,7 +123,7 @@ module.exports = {
 
     writerTimeout: _.template([
       'The writer process timed out after <%= timeout %>s.',
-      'We can\'t guarantee the drive was completely written nor verify its data is correct, please try again.'
+      'There was a synchronization issue while flashing your drive, please try again.'
```

Comment:
Just a minor nit, but when we get this timeout-error, can we tell (based on the messages we've received previously) whether we're still in the flashing-stage, or whether we're in the verifying stage? Would it be worth displaying a slightly different error message in each case?